jupyterhub/values.yaml

```diff
@@ -41,7 +41,7 @@ hub:
   extraVolumeMounts: []
   image:
     name: jupyterhub/k8s-hub
-    tag: generated-by-chartpress
+    tag: 'ac54cf18'
```

Comment:
Undo these changes that come from running `chartpress`

Reply:
- [x] Resolved